ES/ESmy020.xml

```diff
@@ -36,305 +36,277 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                    <q xml:lang="gez">
-                      </q></incipit>
-                      <explicit>
-                      <q xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ
-                      </q></explicit>
-                    <note>The text is the end of <ref corresp="#ms_il.6"/>; the leaf is placed in the reversed position,
-                    to be read from <foreign xml:lang="gez">ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</foreign>, <locus target="2va"/>)</note>
+                  <locus from="2ra" to="71rb"/>
+                  <title type="incomplete" ref="LIT1931Mashaf"/>
                   </msItem>
 
-                    <msItem xml:id="ms_il.2">
-                    <locus from="3ra" to="3rb"/>
-                    <title type="incomplete">Absolution and prayers</title>
-                    <incipit>
-                    <q xml:lang="gez">ሃል፡ ሕዝቦ፡ እስመ፡ ኄር፡ ውእቱ፡ ወመፍቃሬ፡ ሰብእ፡
-                    </q>
-                  </incipit>
-                  <explicit>
-                  <q xml:lang="gez">
-                  </q>
-                </explicit>
-                  <note>The same as <ref type="work" corresp="BAVet74#ms_i1.17.1"/>.
-                  The second part of the text <ref target="#ms_il.11">Absolution and prayers</ref>.</note>
-                  </msItem>
 
-                    <msItem xml:id="ms_il.3">
-                      <locus from="3ra" to="3vb"/>
-                      <title type="incomplete" ref="LIT1931Mashaf">Story of release of an impure rich man from
-                      the Hell</title>
-                      <incipit>
-                      <q xml:lang="gez">ወሀሎ፡ ፩፡ ብእሲ፡ ባዕል፡ በአሐቲ፡ ሀገር፡ ዘኢይፈርሆ፡ ለእግዚአብሔር፡</q>
-                    </incipit>
-                    <explicit>
-                    <q xml:lang="gez">ወዕሙቅ፡ ጥቀ፡ ወም</q>
-                  </explicit>
-                    <note>The leaf contains the beginning of <ref target="#ms_il.12"/> (<foreign xml:lang="gez">
-                  ወሀሎ፡ ብእሲ፡ ባዕል፡ በአሐቲ፡ ሀገር፡</foreign>).</note>
-                    </msItem>
-
-                    <msItem xml:id="ms_il.4">
-                      <locus from="4ra" to="4vb"/>
-                      <title type="incomplete" ref="LIT1629Homily">A portion of the Homily of Jacob of Serug</title>
-                      <note>The leaf missing from from <ref target="ms_il.16"/>, placed in the reversed position (the portion
-                      from <foreign xml:lang="gez">ዘይያጥፍኦ፡ መሐትዊሆን፡</foreign>
-                      to <foreign xml:lang="gez">ወሰረይከ፡ ሎቱ፡ ኃጣ</foreign>).</note>
-                      </msItem>
-
-                      <msItem xml:id="ms_il.5">
+                      <msItem xml:id="ms_i1.1">
                       <locus from="5ra" to="6ra"/>
-                      <title type="incomplete" ref="LIT1931Mashaf#Priests"><ref type="work" corresp="LIT1558Matthew">25:14-30</ref>
+                      <title type="complete" ref="LIT1931Mashaf#Priests"><ref type="work" corresp="LIT1558Matthew">25:14-30</ref>
```

Comment:
complete the title for better visualization (Directions to the priests / excerpt from the Gospel of Matthew, Mt 25:14-30) - if you have text within the title element it is ONLY this text that is visualized first, not the label of the linked file